ios/FluentUI/Core/Theme/Tokens/GradientInfo.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import CoreGraphics
+import Foundation
+import SwiftUI
+
+/// Represents a gradient as used by FluentUI.
+@objc public class GradientInfo: NSObject {
+    /// Initializes a gradient to be used in Fluent.
+    ///
+    /// - Parameters:
+    ///   - colors: The color of the shadow for shadow 1.
+    ///   - startPoint: The blur of the shadow for shadow 1.
+    ///   - endPoint: The horizontal offset of the shadow for shadow 1.
+    public init(colors: [DynamicColor],
+                startPoint: CGPoint,
+                endPoint: CGPoint) {
+        self.colors = colors
+        self.startPoint = startPoint
+        self.endPoint = endPoint
+    }
+
+    /// The array of colors to apply to this linear gradient.
+    public let colors: [DynamicColor]
```

Comment:
looking at CAGardientlayer documentation should we also have locations as optional thing?

Reply:
SwiftUI's `LinearGradient` _does_ require locations, so I'm inclined to side with the pickier of the two.

Reply:
Oh I see what you mean -- adding it as an additional property. Yep okay I can rework that to be added -- the API for SwiftUI is a little different, but I can unify the two.

Reply:
because this gradient struct needs to work for uikit stuff too so....